src/SWP/Bundle/MultiTenancyBundle/Command/CreateOrganizationCommand.php

```diff
@@ -0,0 +1,149 @@
+            ])
+            ->setHelp(
+                <<<'EOT'
+                The <info>%command.name%</info> command creates a new organization.
+EOT
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $name = $input->getArgument('name');
+        $disabled = $input->getOption('disabled');
+
+        $default = $input->getOption('default');
+        if ($default) {
+            $name = OrganizationInterface::DEFAULT_NAME;
+        }
+
+        $organization = $this->getOrganizationRepository()->findByName($name);
+
+        if (null !== $organization) {
+            throw new \InvalidArgumentException(sprintf('"%s" organization already exists!', $name));
+        }
+
+        $organization = $this->createOrganization($name, $disabled);
+
+        $this->getObjectManager()->persist($organization);
+        $this->getObjectManager()->flush();
+
+        $output->writeln(
+            sprintf(
+                'Organization <info>%s</info> has been created and <info>%s</info>!',
+                $name,
+                $organization->isEnabled() ? 'enabled' : 'disabled'
+            )
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        $this->askAndValidateInteract($input, $output, 'name');
+    }
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     * @param string          $name
+     */
+    protected function askAndValidateInteract(InputInterface $input, OutputInterface $output, $name)
+    {
+        $default = $input->getOption('default');
+
+        if (!$input->getArgument($name) && !$default) {
+            $argument = $this->getHelper('dialog')->askAndValidate(
+                $output,
+                '<question>Please enter '.$name.':</question>',
+                function ($argument) use ($name) {
+                    if (empty($argument)) {
+                        throw new \RuntimeException('The '.$name.' can not be empty');
+                    }
+
+                    return $argument;
+                }
+            );
+
+            $input->setArgument($name, $argument);
+        }
+    }
+
+    /**
+     * @param string $name
+     * @param bool   $disabled
+     *
+     * @return OrganizationInterface
+     */
+    protected function createOrganization($name, $disabled)
+    {
+        $organizationFactory = $this->getContainer()->get('swp.factory.organization');
+        /** @var OrganizationInterface $organization */
+        $organization = $organizationFactory->createWithCode();
+        $organization->setName($name);
+        $organization->setEnabled(!$disabled);
+
+        return $organization;
+    }
+
+    /**
+     * @return ObjectManager
+     */
+    protected function getObjectManager()
```

Comment:
I'm not so keen on these one line functions, especially if they are only used in one other function.